File: pbworkspacewcl/src/app/linee-finanziamento/commons/dto/comune.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

export class Comune {
    constructor(
        public descComune: string,
        public descNazione: string,
        public descProvincia: string,
        public descRegione: string,
        public flagNazioneItaliana: string,
        public idComune: string,
        public idNazione: string,
        public idProvincia: string,
        public idRegione: string
    ) { }
}

File: pbweberog/pbweberog-jar/src/main/java/it/csi/pbandi/pbweberog/dto/datiprogetto/Comune.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbweberog.dto.datiprogetto;


public class Comune implements java.io.Serializable {

	private Long idComune;
	private String codIstatComune;
	private String descrizione;
	private static final long serialVersionUID = 1L;
	
	


	public Long getIdComune() {
		return idComune;
	}




	public void setIdComune(Long idComune) {
		this.idComune = idComune;
	}




	public String getCodIstatComune() {
		return codIstatComune;
	}




	public void setCodIstatComune(String codIstatComune) {
		this.codIstatComune = codIstatComune;
	}




	public String getDescrizione() {
		return descrizione;
	}




	public void setDescrizione(String descrizione) {
		this.descrizione = descrizione;
	}




	public Comune() {
		super();

	}

}
